content/en/docs/distributions/gke/deploy/deploy-cli.md

```diff
@@ -1,27 +1,25 @@
 +++
-title = "Deploy using kubectl and kpt"
+title = "Deploy Kubeflow cluster"
 description = "Instructions for using kubectl and kpt to deploy Kubeflow on Google Cloud"
-weight = 4
+weight = 5
 +++
 
 This guide describes how to use `kubectl` and [kpt](https://googlecontainertools.github.io/kpt/) to
 deploy Kubeflow on Google Cloud.
 
-## Before you start
+## Deployment steps
+
+### Prerequisites
 
 Before installing Kubeflow on the command line:
 
 1. You must have created a management cluster and installed Config Connector.
 
-   * If you don't have a management cluster follow the [instructions](../management-setup/)
+   * If you don't have a management cluster follow the [instructions](/docs/gke/deploy/management-setup/)
```

Comment:
Interestingly the link still works without `distributions` path. But I will update them to include `distributions`, the reason of using absolute path is that it is easy to do batch update using IDE if we move files going forward. I slightly prefer absolute path.

Reply:
That's because we added redirection rules from the old location to the new one, but when writing a link, we should use its current address (better for SEO).
+1 to use absolute paths.